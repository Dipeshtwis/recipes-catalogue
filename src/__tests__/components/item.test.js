import React from 'react';
import renderer from 'react-test-renderer';
import item from '../../components/Item';

it('renders item component correctly', () => {
  const item = renderer.create(<item />).toJSON();
  expect(item).toMatchSnapshot();
});