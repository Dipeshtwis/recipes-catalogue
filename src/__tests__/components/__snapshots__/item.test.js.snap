// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders item component correctly 1`] = `<item />`;
